### ISSUE TITLE
NaN or inf in Input Tensor after 1st epoch? T5Model

### ISSUE THREAD

**Issue by tshrjn** [ThilinaRajapakse/simpletransformers#1087]
**Describe the bug**
When training T5Model, I start getting the `Warning: NaN or inf in Input Tensor` but only after successful training of the 1st epoch. Since the eval & training data are okay for the 1st epoch (eval steps < no of steps in epoch), what could be causing such a warning? Debugging recommendations, some issue might be silently creeping in.

In fact, it does creep in & makes the loss Nan after a few epochs.

**Desktop:**
 - Ubuntu 18.04 with 2080Ti (CUDA Version: 10.2)

**Comment by ThilinaRajapakse** [comment]
One possibility I can think of is that the model is being trained for too many steps which can "break" transformer models. Some of the weights might be messed up to the point that the model is generating extremely large values.
